dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -103,7 +103,10 @@ public OrganizationResource(SessionFactory sessionFactory) {
     @Operation(operationId = "getApprovedOrganizations", summary = "List all available organizations.", description = "List all organizations that have been approved by a curator or admin, sorted by number of stars.")
     public List<Organization> getApprovedOrganizations() {
         List<Organization> organizations = organizationDAO.findApprovedSortedByStar();
-        organizations.stream().forEach(org -> Hibernate.initialize(org.getAliases()));
+        organizations.stream().forEach(org -> {
+            Hibernate.initialize(org.getAliases());
+            org.setCollectionsLength(org.getCollections().size());
```

Comment:
How many additional SQL queries does this cause? If it's 0, then it's fine, if it's the number of collections, then we might want to think about a dedicated query.

Reply:
Discussed with Charles on Slack. There's one SQL statement per organization, so will leave it as it is for now, might cause issues when the number of organization grows